provider/pkg/await/await_test.go

```diff
@@ -501,10 +495,9 @@ func Test_Deletion(t *testing.T) {
 				name:    "foo",
 				objects: []runtime.Object{validPodUnstructured},
 				inputs:  withSkipAwait(validPodUnstructured),
-				outputs: validPodUnstructured,
+				outputs: withSkipAwait(validPodUnstructured),
```

Comment:
The test was previously using `c.Inputs` to check whether to skip the await. We're now using `c.Output` which is equivalent for the deletion code path.

Reply:
Fair! I'll try switching this back - I think I ran into a failure somewhere, but it might have been due to a quirk of the test or something else.